nan.h

```diff
@@ -1393,6 +1393,19 @@ class NanCallback {
     NanDisposePersistent(handle);
   }
 
+  bool operator==(const NanCallback &other) const {
+    v8::Local<v8::Value> a = NanNew(handle)->Get(kCallbackIndex);
+    v8::Local<v8::Value> b = NanNew(other.handle)->Get(kCallbackIndex);
+    if (a->IsUndefined() || b->IsUndefined()) {
+      return a->IsUndefined() && b->IsUndefined();
+    }
+    return a->StrictEquals(b);
```

Comment:
Now that you're using StrictEquals() you don't need the IsUndefined() checks any more.